examples/plot_Dreyer_clf_scores_vs_subj_info.py

```diff
@@ -0,0 +1,60 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sb
+from mne.decoding import Vectorizer
+from pyriemann.estimation import Covariances
+from pyriemann.spatialfilters import CSP
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
+from sklearn.pipeline import make_pipeline
+
+import moabb
+from moabb.datasets import Dreyer2023A, Dreyer2023B, Dreyer2023C
+from moabb.evaluations import WithinSessionEvaluation
+from moabb.paradigms import MotorImagery
+
+
+dreyer2023 = Dreyer2023A()
```

Comment:
```suggestion
dreyer2023 = Dreyer2023A()
dreyer2023.subject_list = dataset.subject_list[:6]
```
Generating documentation uses lots of computational ressources, both for CPU and network bandwidth. We try to limit as much as possible the datasets downloaded for generating docs (they are shared between the examples). As the user's profile included in this dataset could give some interesting insight, we could consider to add this dataset in the documentation, but it should use only a selection of subjects to keep the bandwidth usage within reasonable limits. Could you adapt the above line to select only a representative individual for your analysis below. If you could keep the number of subjects between 4 and 8, it will really lower the doc building time.
Cherry picking subjects for the documentation is allowed ^^

Reply:
Thanks! I've selected 4 subjects (to include all combinations of participants' and experimenters' genders). However, since data of all subjects are zipped they all need to be downloaded together, is there some solution for this?

Reply:
No unfortunately.
The only possibility I see is to make this example a static one, that is you change the name of the file into `Dreyer_clf_scores_vs_subj_info.py` (without the `plot_` prefix) and Sphinx won't execute this example, only display the code. If you want, you could generate a figure on your computer and upload it in a new folder `examples/images/`, then you could include the resulting image from your example code (see for example [this page](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html) or [that page](https://stackoverflow.com/questions/25866102/how-do-we-embed-images-in-sphinx-docs)).